udata/core/tags/api.py

```diff
@@ -24,5 +24,5 @@ def get(self):
         '''Suggest tags'''
         args = parser.parse_args()
         q = normalize(args['q'])
```

Comment:
I don't think we should normalize tags in tag suggest. It returns unrelevant results by changing the query from `'co'` to `''`.

Reply:
This seems to be a wanted behaviour as the function returns a value of `''` if the query is inferior to `current_app.config['TAG_MIN_LENGTH']`
```
if len(value) < MIN_TAG_LENGTH:
    value = ''
```

Reply:
Agreed, but I don't think we want this normalization in the case of suggest.